### ISSUE TITLE
Gnome platform runtime version 40 has reached end of life

### ISSUE THREAD

**Issue by parcimonic** [flathub/com.github.babluboy.bookworm#18]
```text
Info: runtime org.gnome.Platform branch 40 is end-of-life, with reason:
   The GNOME 40 runtime is no longer supported as of March 21, 2022. Please ask your application developer to migrate to a supported platform.
Info: applications using this runtime:
   com.github.babluboy.bookworm

Info: runtime org.freedesktop.Platform.GL.default branch 20.08 is end-of-life, with reason:
   org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
Info: applications using this extension:
   com.github.babluboy.bookworm
```

I've tried to update the runtime (and the "base" `io.elementary.BaseApp`), but the build always fails. I've tried changing Poppler versions, etc, nothing worked.

Gnome 41 (also EOL), 42 and 43 failed to build. Mixing that with different versions of Elementary base also failed. Mixing all of those with different Poppler versions also failed.

My conclusion is that the app need code updates, unless someone else has a patch that can be applied in this repo.

**Comment by babluboy** [comment]
> ```
> Info: runtime org.gnome.Platform branch 40 is end-of-life, with reason:
>    The GNOME 40 runtime is no longer supported as of March 21, 2022. Please ask your application developer to migrate to a supported platform.
> Info: applications using this runtime:
>    com.github.babluboy.bookworm
> 
> Info: runtime org.freedesktop.Platform.GL.default branch 20.08 is end-of-life, with reason:
>    org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
> Info: applications using this extension:
>    com.github.babluboy.bookworm
> ```
> 
> I've tried to update the runtime (and the "base" `io.elementary.BaseApp`), but the build always fails. I've tried changing Poppler versions, etc, nothing worked.
> 
> Gnome 41 (also EOL), 42 and 43 failed to build. Mixing that with different versions of Elementary base also failed. Mixing all of those with different Poppler versions also failed.
> 
> My conclusion is that the app need code updates, unless someone else has a patch that can be applied in this repo.

Yes. Working on code updates to move bookworm  to gnome44, gtk 4 which requires latest version of gdk and granite. I might have to abandon poppler if it does not work with these libraries and use some other library to convert pdf to html

**Comment by yakushabb** [comment]
Fixed
